app_dart/lib/src/service/scheduler.dart

```diff
@@ -142,7 +142,11 @@ class Scheduler {
         policy = GuaranteedPolicy();
       }
       final int? priority = await policy.triggerPriority(task: task, datastore: datastore);
-      if (priority != null) {
+      // Skip scheduling `bringup: true` targets. They may be newly created and
+      // their corresponding LUCI builder configs may not be ready yet considering we
+      // disabled the `ci_yaml roller` backfill. Existing `bringup: true` targets
+      // can rely on backfiller to get tasks scheduled and executed.
+      if (priority != null && !target.value.bringup) {
```

Comment:
Good catch!

Reply:
What is the need for the priority not being null check? What if it is null?

Reply:
For batch scheduler policy, if there is a running (in progress) task, it will return null and the tot ones will not be scheduled right away.